client/src/components/AnswerCard.js

```diff
@@ -3,13 +3,13 @@ import styled from '@emotion/styled';
 
 const AnswerContainer = styled.button`
   display: flex;
-  outline: none;
-  border: 4px solid ${props => props.theme.colors.text1};
   justify-content: center;
+  outline: none;
   margin: 10px;
   width: 160px;
   height: 90px;
```

Comment:
I'm not sure about fixed `width` & `height`, might be worth considering a separate overhaul for responsiveness if you find the time to do so ;)

Reply:
Yeah, I was thinking about that too